### DISCUSSION TITLE
Pre commit check on committing any changes

### DISCUSSION THREAD

**Discussion by 87prashant** [ParabolInc/parabol#6749] [Q&A] [answered]
I am started getting a Pre-commit check on committing any changes which I did not get earlier
Does anyone know why it is happening?

![Screenshot 2022-06-14 230246](https://user-images.githubusercontent.com/106697681/173641179-35f85256-0470-4969-b8c5-efa160d828ac.png)

**Comment by Dschoordsch** [comment, marked as answer]
Yes, it's because we're installing a hook via [husky](https://github.com/typicode/husky) for this repository https://github.com/ParabolInc/parabol/blob/c00a6799e0f814717a66e4aef94a7bb96ecc9298/package.json#L48
It runs some linting checks locally.

**Reply by 87prashant** [reply]
@Dschoordsch Thanks
Do I have to do anything because it is showing errors in files that I have not changed? like `EmailPasswordAuthForm.tsx`
